ai-platform-unified/notebooks/unofficial/matching_engine/matching_engine_for_indexing.ipynb

```diff
@@ -725,8 +725,9 @@
       },
       "outputs": [],
       "source": [
+        "from google.protobuf import *",
```

Comment:
from google.protobuf import struct_pb2 is in the "Import libraries and define constants" section to fit into the notebook sample template.  I think the notebook instance might logged out so that the import is missing.

Reply:
Right, I think that with the 2-3 polling queries the nb often times out and the libraries are lost.